apollo-core/src/main/java/com/ctrip/framework/apollo/core/enums/EnvUtils.java

```diff
@@ -18,31 +18,40 @@
 
 import com.ctrip.framework.apollo.core.utils.StringUtils;
 
+/**
+ * A utility class for the {@link Env} enum.
+ * <p>
+ * The class provides simple functionalities that extend the capabilities of {@link Env}
+ *
+ * @author Diego Krupitza(info@diegokrupitza.com)
+ */
 public final class EnvUtils {
-  
+
+  /**
+   * Transforms a given String to its matching {@link Env}
+   *
+   * @param envName the String to convert
+   * @return the matching {@link Env} for the given String
+   */
   public static Env transformEnv(String envName) {
     if (StringUtils.isBlank(envName)) {
       return Env.UNKNOWN;
     }
-    switch (envName.trim().toUpperCase()) {
-      case "LPT":
-        return Env.LPT;
-      case "FAT":
-      case "FWS":
```

Comment:
Seems you missed this case

Reply:
This is now resolved by adding a special case check before